pkg/app/utils.go

```diff
@@ -21,16 +21,25 @@ import (
 )
 
 const (
-	dbTemplateURI = "https://raw.githubusercontent.com/openshift/origin/v3.11.0/examples/db-templates/%s-%s-template.json"
+	dbTemplateURI = "https://raw.githubusercontent.com/openshift/origin/%s/examples/db-templates/%s-%s-template.json"
 	// PersistentStorage can be used if we want to deploy database with Persistent
 	PersistentStorage storage = "persistent" // nolint:varcheck
 
 	// EphemeralStorage can be used if we don't want to deploy database with Persistent
 	EphemeralStorage storage = "ephemeral"
+	// TemplateVersionOCP3_11 stores version of db template 3.11
+	TemplateVersionOCP3_11 DBTemplate = "release-3.11"
+	// TemplateVersionOCP4_4 stores version of db template 4.4
+	TemplateVersionOCP4_4 DBTemplate = "release-4.4"
+	// TemplateVersionOCP4_5 stored version of db template 4.5
+	TemplateVersionOCP4_5 DBTemplate = "release-4.5"
```

Comment:
Can we make it more generic. So version passed with `./build/integration-test.sh openshift <version>` could be any string. And we use that string to generate `dbTemplateURI`, if version is invalid, the test will anyway fail. This saves our efforts of maintaining support for several versions.

Reply:
I dont think we should allow any version to be passed to `integration-test.sh` because 
1. It's better that we let someone know that this version is not supported than let him wait for the time test is running and then after test fails he will have to figure out that this version is not supported.
2. This will help us as well, in order to keep track of the version that we have tested with and are supported.
what do you think

Reply:
Sounds good